src/rest.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_REST_H
+#define BITCOIN_REST_H
+
+#include <string>
+
+enum class RetFormat {
```

Comment:
That's a good idea. I've scripted-diff renamed `RetFormat` to `RESTResponseFormat`, since this enum is currently only used to describe response format.